tests/main/fv3core/test_grid.py

```diff
@@ -13,6 +12,7 @@
     Z_DIM,
     Z_INTERFACE_DIM,
 )
+from ndsl.dsl.gt4py import I, J, K
```

Comment:
I'd rather `import ndsl.dsl.gt4py as gt` or something to maintain clarity in the code

Reply:
Your codebase, your rules. But what are you afraid of clashing with?

Also, unrelated, but having Z_DIM, Float and such in a different import path than the frontend stencils irks me. We might have to iterate on all those imports

Reply:
it seems aesthetically displeasing to have them running about without some kind of nametag

But you're right about having all of the imports coming from different paths, we definitely should clean that up